index.js

```diff
@@ -26,7 +26,10 @@ module.exports = opts => {
 			file.contents = Buffer.from(res.css);
 
 			if (res.map && file.sourceMap) {
-				applySourceMap(file, res.map.toString());
+				const map = res.map.toJSON();
+				map.file = file.relative;
+				map.sources = map.sources.map(() => file.relative);
```

Comment:
changing all sources to be the same than `map.file` ? Is that expected ?

Reply:
There is only one source file, which is `file`. Maybe more obvious would be to set `map.sources = [file.relative]`.